### PR TITLE
task-5: import file service

### PR DESCRIPTION
**Pull request by AntonVolyk**
Done:
1 - File serverless.yml contains configuration for importProductsFile function
3 - The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
4 - Frontend application is integrated with importProductsFile lambda
5 - The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda

Additional (optional) tasks:
+1 (for JS only) - async/await is used in lambda functions
+1 (All languages) - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into a new folder in the same bucket called parsed, and then deleted from uploaded folder)

FE PR: https://github.com/AntonVolyk/shop-angular-cloudfront/pull/4
getSignedUrl: https://0pf7ai4rr7.execute-api.eu-west-1.amazonaws.com/dev/import?name=test
FE link:  d2vqi3n9ulcbuk.cloudfront.net

### CODE REVIEW

File: product-service/environment.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const environment = {
```
Comment (Mrinalaot) [inline product-service/environment.ts:1]: Please don't checkin env related files to git. Move then to .env and place in .ignore

File: import-service/src/functions/import-file-parser/handler.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    })
+                    .on('end', async () => {
+                        try {
+                            await s3.copyObject({
+                                Bucket: BUCKET_NAME,
+                                CopySource: `${BUCKET_NAME}/${record.s3.object.key}`,
+                                Key: 'parsed'
+                            }).promise();
+
+                            await s3.deleteObject({
```
Comment (Mrinalaot) [inline import-service/src/functions/import-file-parser/handler.ts:35]: Better to segregate to methods and use it here like read, copy and delete 3 different functionalities

**Comment by Mrinalaot** [comment]
Rest looks good. Completed all task **Score : 8**

**Comment by VitaliiKopylov** [comment]
Everything looks good, but leaking of secret keys can lead to problems.
**Score : 7** (No unit-tests)

**Pull request merged**